content/docs/oracle/run-oracle/_index.md

```diff
@@ -121,21 +205,60 @@ Additional data adaptors for any external data source can be developed using the
 - Guide for how _To write a new plugin_ using the template in [`/plugins/README`<i class='fas fa-external-link-alt'></i>](https://github.com/clearmatics/autonity-oracle/tree/master/plugins#readme).
 
 
-<!--
 ## Run Autonity Oracle Server as Docker image {#run-docker}
 
-TODO
--->
-<!--  
-## Run Autonity Oracle Server as Linux daemon service {#run-daemon}
+- Ensure that the Autonity Oracle Server [Docker image](/oracle/install-oracle/#install-docker) has been installed.
+
+1. Enter your working directory for autonity oracle server.
+
+2. Edit your oracle server config file `plugins-conf.yml` to specify the `name` and `key` for each plugins you are using. 
+3. Set the Docker configuration and the arguments for running Autonity Oracle Server and connecting to the Autonity Go Client it is serving.
+
+   ```bash
+   docker run \
+        -t -i \
+        --volume $<PATH_TO_ORACLE_KEY_FILE>:/autoracle/oracle.key \
+        --volume $<PATH_TO_PLUGINS_FILE>:/autoracle/plugins-conf.yml \
+        --name oracle-server \
+        --rm \
+        ghcr.io/clearmatics/autonity-oracle:v0.1.1-a2d406f \
+        -oracle_key_file="/autoracle/oracle.key" \
```

Comment:
Please update the flags' name again.

Reply:
Done, resolved in commit 7b5a773d9d7c2590fad714687b3a1fcdea79e704